tests/test_more.py

```diff
@@ -56,6 +57,17 @@ def test_querying_array_field_contains(self):
         self.assertFalse(
             Hand.objects.filter(cards__contains=[jack_of_spades]).exists())
 
+    def test_generate_migrations(self):
+        """Test deconstruction of composite type as a base field"""
+        field = Hand._meta.get_field('cards')
+        text, imports = MigrationWriter.serialize(field)
+        self.assertEqual(text,
+                         'django.contrib.postgres.fields.ArrayField'
```

Comment:
Ah, serves me right for only running tox with one environment. Should be fixed now.